src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/HSDB.java

```diff
@@ -78,10 +78,10 @@ public static void main(String[] args) {
   private JInternalFrame consoleFrame;
   private WorkerThread workerThread;
   // These had to be made data members because they are referenced in inner classes.
-  private String pidText;
   private int pid;
   private String execPath;
   private String coreFilename;
+  private String remoteMachineName;
```

Comment:
`debugServerName` or `remoteDebugServerName`

Reply:
Rename to `debugServerName` in new commit.